FrascatiPlugin.php

```diff
@@ -45,10 +45,35 @@ public function register($category, $path, $mainContextId = null)
         if ($success && $this->getEnabled()) {
             // Add hook
             Hook::add('API::vocabs::external', $this->setData(...));
+            Hook::add('Form::config::after', $this->addVocabularyToKeywordsField(...));
+
         }
         return $success;
     }
 
+     public function addVocabularyToKeywordsField($hookName, $args) {
+
+        if ($args[0]["id"] == 'metadata') {
+            // Find the keywords field
+            foreach ($args[0]["fields"] as $key => $field) {
+                if ($field["name"] == "keywords") {
```

Comment:
(As a general practice, it's better to use `'` for strings unless you need string interpolation. Here and below.)
```suggestion
                if ($field['name'] == 'keywords') {
```